apiserver/Dockerfile

```diff
@@ -0,0 +1,36 @@
+#
+# Dockerfile for building APISERVER
+#
+
+# Build the APISERVER using phusion base image
+
+FROM phusion/baseimage:master-amd64
+
+# Installing nodejs 
+RUN curl -sL https://deb.nodesource.com/setup_16.x | bash -
+RUN apt-get update && apt-get install nodejs -y
+
+# Copying public keys to get SSH access to this Container
+COPY *.pub /tmp/
+RUN cat /tmp/*.pub >> /root/.ssh/authorized_keys && rm -f /tmp/*.pub
+
+# Enabling SSH service
+RUN rm -f /etc/service/sshd/down
+RUN /etc/my_init.d/00_regen_ssh_host_keys.sh
```

Comment:
This might be better done right after line 7; that way the host keys are unlikely to change. I'd do it first, in fact. I think line 19 can move to right after line 7.

Reply:
Yes. As you advised, I've made the changes required.